src/builtins/set_color.rs

```diff
@@ -167,7 +167,9 @@ pub fn set_color(parser: &Parser, streams: &mut IoStreams, argv: &mut [&wstr]) -
         }
     }
     if let Some(bg) = bg {
-        if !bg.is_normal() {
+        if bg.is_normal() {
+            outp.write_command(DefaultBackgroundColor);
```

Comment:
to resolve the inconsistency for foreground, we could add `set_color --foreground=normal` which only resets the foreground color. Doesn't need to be added now though